src/main/java/com/commercetools/sync/products/templates/beforeupdatecallback/SyncSingleLocale.java

```diff
@@ -76,13 +76,15 @@
      * @param locale       the locale value to only compare and map the update action to accordingly.
      * @return an optional containing the mapped update action or empty value if an update action is not needed.
      */
+    @Nonnull
     private static Optional<UpdateAction<Product>> filterSingleLocalization(
         @Nonnull final UpdateAction<Product> updateAction,
         @Nonnull final ProductDraft newProductDraft,
         @Nonnull final Product oldProduct,
-        @Nonnull final ProductType productType,
+        //@Nonnull final ProductType productType,
```

Comment:
is this ok? we can get this commented code from history.

Reply:
true! :)

Reply:
e6376dc